### PR TITLE
French translation adjusted to the recent changes

### PR DESCRIPTION
**Pull request by lolodomo**
Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

File: mobile/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -27,11 +29,14 @@
...
     <string name="settings_openhab_password">Mot de passe</string>
+    <string name="settings_openhab_sslclientcert">Certificat SSL client</string>
+    <string name="settings_openhab_sslclientcert_howto">Certificat SSL client - comment faire</string>
+    <string name="settings_openhab_sslclientcert_howto_summary">Taper ici pour avoir de l\'aide sur comment configurer l\'authentification par certificat SSL</string>
     <string name="settings_openhab_screentimeroff">Désactiver l\'extinction d\'écran</string>
-    <string name="settings_openhab_screentimeroff_summary">Désactive l\'extinction d\'écran lorsque l\'app openHAB fonctionne</string>
+    <string name="settings_openhab_screentimeroff_summary">Désactiver l\'extinction d\'écran lorsque l\'app openHAB fonctionne</string>
     <string name="settings_openhab_demomode">Mode démo</string>
     <string name="settings_openhab_demomode_summary">Lancer l\'app openHAB en mode démo</string>
     <string name="settings_openhab_theme">Thème app openHAB</string>
```
Comment (mueller-ma) [inline mobile/src/main/res/values-fr/strings.xml]: Maybe you want to remove "app openHAB" from this string.

Comment (lolodomo) [reply mobile/src/main/res/values-fr/strings.xml]: Why ? "openHAB app" is now the official name in the store.

Comment (mueller-ma) [reply mobile/src/main/res/values-fr/strings.xml]: I think it was changed to "Theme" in English and to "Design" in German.

**Comment by digitaldan** [comment]
@lolodomo since I don't speak/read French, I will take your word on this :-)

**Pull request merged**